pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["setuptools >=42", "setuptools_scm[toml] >=3.4", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+write_to = "pytest_localserver/version.py"
```

Comment:
You mean like `__version__.py`? I didn't do that because I figured underscore names might be reserved for special purposes and I didn't want to introduce a conflict, but if it's safe to use `__version__.py` for this purpose, that works for me.

What's the purpose of importing it into `__init__.py`?

Reply:
`_version.py`

`from. _version import version as versucht ` in `__init__.py` to make it public

Reply:
Ah, I see what you mean. Yeah, I could do that.